README.md

```diff
@@ -1,3 +1,48 @@
 [![Build Status](https://api.travis-ci.org/alphagov/paas-cf.svg)](https://travis-ci.org/alphagov/paas-cf)
 
 # paas-cf
+
+This repository contains [Concourse](http://concourse.ci/) pipelines and related [Terraform](https://terraform.io/) and [BOSH](https://bosh.io/) manifests that allow provisioning of [CloudFoundy](https://www.cloudfoundry.org/) on AWS. It consists of two chains:
+- A concourse pipeline to provision CloudFoundry on AWS using Concourse, Terraform and BOSH
+- A concourse pipeline to destroy previously provisioned environment.
+
+## Operation
```

Comment:
In that case:

 * I would rather just plaing list the pipelines and refer to the central documentation. 
 * move all this "Operation" steps to describe each one of the pipelines deployed with the scripts listed in "Usage / Build"

I said so because this Operation section will get really complicated as we add more pipelines, and will quickly diverge from the actual code.